### PR TITLE
OSDOCS 12369: Updated cert-manager release note for 1.14.1 version

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.14-4.17

Issue: https://issues.redhat.com/browse/OSDOCS-12369

Link to docs preview: https://83750--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-release-notes.html#cert-manager-operator-release-notes-1-14-1

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 1]
```diff
@@ -12,6 +12,24 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-14-1"]
+== {cert-manager-operator} 1.14.1
+
+Issued: 2024-10-17
```
Comment (swghosh) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:18]: Planned release date is 04th November, 2024 so ti's can be updated accordingly.

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 2]
```diff
@@ -12,6 +12,24 @@ These release notes track the development of {cert-manager-operator}.
...
+
+The following advisory is available for the {cert-manager-operator} 1.14.1:
+
+* link:https://errata.devel.redhat.com/advisory/140037[RHEA-2024:140037]
```
Comment (swghosh) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:22]: IIRC, this has to be an access.redhat.com link for external customers to be able access the info.

**Review by swghosh — APPROVED** [review]
generally LGTM, thanks!

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc (hunk 1 above)
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:18]: Ack

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc (hunk 2 above)
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:22]: Yep, once the advisory is done, I will update the link.

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 3]
```diff
@@ -12,6 +12,24 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-14-1"]
```
Comment (skopacz1) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:15]: All IDs except for the first one in an assembly need a context, per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings):
```suggestion
[id="cert-manager-operator-release-notes-1-14-1_{context}"]
```

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 4]
```diff
@@ -12,6 +12,24 @@ These release notes track the development of {cert-manager-operator}.
...
+
+Issued: 2024-11-04
+
+The following advisory is available for the {cert-manager-operator} 1.14.1:
+
+* link:https://access.redhat.com/errata/RHEA-2024:8255[RHEA-2024:8255]
+
+Version `1.14.1` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.14.7`. For more information, see the link:https://cert-manager.io/docs/releases/release-notes/release-notes-1.14/#v1147[cert-manager project release notes for v1.14.7].
+
+[id="cert-manager-operator-1-14-1-CVEs"]
```
Comment (skopacz1) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:26]: Same comment as before:
```suggestion
[id="cert-manager-operator-1-14-1-CVEs_{context}"]
```

**Review by skopacz1 — COMMENTED** [review]
Just two comments, otherwise LGTM

**Pull request closed without merging**